doc/source/index.rst

```diff
@@ -1,13 +1,106 @@
-.. raw:: html
-
-    <form class="bd-search align-items-center" action="search.html" method="get">
-      <input type="search" class="form-control search-front-page" name="q" id="search-input" placeholder="&#128269; Search the docs ..." aria-label="Search the docs ..." autocomplete="off">
-    </form>
+:html_theme.sidebar_secondary.remove:
```

Comment:
this is what removes the right sidebar from the page, it needs to be set on a page by page basis.